Resources/Prototypes/_Mono/Entities/SpaceArtillery/SpaceArtillery/BallisticArtillery.yml

```
@@ -82,8 +87,71 @@
+  - type: ExtensionCableReceiver
+  - type: ProjectileBatteryAmmoProvider
+    proto: ShipMediumPlasmaProjectile
+    fireCost: 500
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 600
+      behaviors:
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)